core/src/main/java/ch/cyberduck/core/Local.java

```diff
@@ -426,25 +426,38 @@ public String toURL() {
         return String.format("file:%s", path);
     }
 
+
     public InputStream getInputStream() throws AccessDeniedException {
-        return this.getInputStream(path);
+        return getInputStream(path);
     }
 
-    protected InputStream getInputStream(final String path) throws LocalAccessDeniedException {
+    public InputStream getInputStream(final String path) throws AccessDeniedException {
```

Comment:
This can remain `protected` or even `private` I guess.

Reply:
fixed